src/taker/api.rs

```diff
@@ -106,6 +106,62 @@ enum TakerPosition {
     LastPeer,
 }
 
+/// Information about individual maker fees in a swap
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct MakerFeeInfo {
+    /// Index of maker(starts with zero)
+    pub maker_index: usize,
+    /// Maker Addresses (Onion:Port)
+    pub maker_address: String,
+    /// The fixed Base Fee for each maker
+    pub base_fee: f64,
+    /// Dynamic Amount Fee for each maker
+    pub amount_relative_fee: f64,
+    /// Dynamic Time Fee(Decreases for subsequent makers) for each maker
+    pub time_relative_fee: f64,
+    /// All inclusive fee for each maker
+    pub total_fee: f64,
```

Comment:
What is this total fee? Total fee is dependent on the amount. How do we know the total fee in this struct?